dependencies/tekton-results/kustomization.yml

```diff
@@ -3,7 +3,7 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
   # renovate: datasource=git-refs depName=https://github.com/tektoncd/results versioning=semver
-  - https://infra.tekton.dev/tekton-releases/results/previous/v0.16.0/release.yaml
+  - https://infra.tekton.dev/tekton-releases/results/previous/v0.17.0/release.yaml
```

Comment:
The file does not exist (yet?).
The [releases page](https://github.com/tektoncd/results/releases) does point to the file in another location. If this does not resolve itself in a bit, we can consider changing the reference we're using.